internal/acceptance/openstack/objectstorage/v1/objects_test.go

```diff
@@ -22,19 +22,21 @@ import (
 var numObjects = 2
 
 func TestObjects(t *testing.T) {
+	numObjects := numObjects + 1
 	client, err := clients.NewObjectStorageV1Client()
 	if err != nil {
 		t.Fatalf("Unable to create client: %v", err)
 	}
 
 	// Make a slice of length numObjects to hold the random object names.
 	oNames := make([]string, numObjects)
-	for i := 0; i < len(oNames); i++ {
-		oNames[i] = tools.RandomString("test-object-", 8)
+	for i := 0; i < len(oNames)-1; i++ {
+		oNames[i] = "test-object-" + tools.RandomFunnyString(8)
 	}
+	oNames[len(oNames)-1] = "test-object-with-/v1/-in-the-name"
```

Comment:
There were two test cases. Now there are three: the last one tests that we correctly split the object path in `CreateTempURL`.